File: p005/p5.cpp
#include <iostream>

using namespace std;

bool isDivisible(int n) {
	for (int j = 11; j < 21; ++j) {
		if (n % j != 0)
			return false;
	}
	return true;
}

int main() {
	for (unsigned int i = 1; i < 4000000000; ++i) {
		if (isDivisible(i)) {
			cout << i << endl;
			break;
		}
	}
	
	return 0;
}

File: p005/p5.rs
fn isDivisible(num : int) -> bool {
	for i in range(11, 21) {
		if num % i != 0 {
			return false;
		}
	}
	true
}

fn main() {
	let mut n = 1;
	loop {
		if isDivisible(n) {
			println!("{}", n);
			break;
		}
		n += 1;
	}	
}
